server/src/test/java/org/elasticsearch/cluster/routing/allocation/decider/MockDiskUsagesIT.java

```diff
@@ -426,4 +513,30 @@ private MockInternalClusterInfoService getMockInternalClusterInfoService() {
         return (MockInternalClusterInfoService) internalCluster().getCurrentMasterNodeInstance(ClusterInfoService.class);
     }
 
+    private Map<String, Integer> getShardCountByNodeId(boolean primary, ShardRoutingState... shardState) {
```

Comment:
We only care about the number of initialising shards on `node1` (and they are all replicas) so I think you can simply use `client().admin().cluster().prepareState().get().getState().getRoutingNodes().node(node1Id).numberOfShardsWithState(ShardRoutingState.INITIALIZING);`